src/main/java/seedu/duke/commands/EmptyCommand.java

```diff
@@ -0,0 +1,27 @@
+package seedu.duke.commands;
+
+import seedu.duke.storage.Storage;
+import seedu.duke.storage.TaskList;
+
+public class EmptyCommand extends Command {
+
+    /**
+     * Empty Command that does nothing.
+     * 
+     * @param taskList the list of Tasks which is being stored.
+     * @param storage  the database where the Tasks are being saved for progression.
+     */
+    public String execute(TaskList taskList, Storage storage) {
+        return "Hi how can I help you?";
+    }
+
+    /**
+     * Checks if the user wants to exit from the application.
+     * 
+     * @return boolean whether the user wants to exit from the application.
+     */
+    @Override
+    public boolean getIsExit() {
+        return false;
+    }
+}
```

Comment:
Similar comments with parent class. Also, overidden classes do not necessarily need javadocs, especially since it is so similar with the parent's docs.